src/app/store/pending.ts

```diff
@@ -1,5 +1,6 @@
 export enum Pending {
   None,
   Active,
+  Success,
```

Comment:
Pending can't be a success.
You use it only for managing the state in the forgot-password component and won't use it anywhere else.